.github/source_plutus_apps.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+echo "::group::plutus-apps setup"
+
+pushd "$WORKDIR" || exit 1
+
+# clone plutus-apps if needed
```

Comment:
comments should start with a capital letter

Reply:
Can be, but we don't follow any particular code style for shell scripts.